umbrella/router.go

```diff
@@ -58,12 +58,11 @@ type funcRoutingInfo struct {
 	Manual      string
 }
 
-// confirmFunctionIsRegistered is designed to eliminate case sensitivity when calling a registered function.
-// EqualFold() is used to determine that the requested function is equal to a registered function under Unicode case-folding.
+// confirmFunctionIsRegistered is designed to return function registry info regardless of case sensitivity.
 func confirmFunctionIsRegistered(reqFunc string, funcReg *map[string]funcRoutingInfo) (string, error) {
-	for key, _ := range funcReg {
+	for key := range funcReg {
```

Comment:
Range is a multi-return statement. This assignment will fail. You need that blank assignment.

Reply:
VS Code/Go extension was telling me the _ was redundant and is interpreted the same as if the _ wasn't there, so I removed it to get rid of the squiggly line. I can add it back.

Reply:
Just touching back on this, I think it is valid without the _ in place of `value`. I think you would do `for _, value` if you just wanted the value and not the key, but if you don't need the value, it can be omitted. 

it is the golang.org `go-lint` package telling me to drop the _ in VS Code, and the range section, here:
http://golangtutorials.blogspot.com/2011/06/control-structures-go-for-loop-break.html